### ISSUE TITLE
Navbar dropdown not working on PHP Desktop v57.0 for Windows

### ISSUE THREAD

**Issue by zikomik** [jgthms/bulma#2644]
Navbar dropdown not working on PHP Desktop v57.0 for Windows.

Dropdown not opening in navbar

**Comment by jgthms** [comment]
Going to need more details.

**Comment by kristwa** [comment]
Dropdown not working on MS Edge (not the Chromium based version) either. Tested on https://bulma.io/documentation/components/navbar/ with Edge 44.18362.329.0.

**Comment by thijsfranck** [comment]
I ran into this issue as well. Here's my workaround:

Normally, when hovering over an element with class `.navbar-item.has-dropdown.is-hoverable`, Bulma assigns `display: block` to any child `.dropdown-menu`. This works fine for Webkit browsers and Firefox. However, in Edge and IE11 (haven't tried older versions), the `display` property would remain at `display: none`.

As a workaround, I added the following to the CSS of my header component:

```
.navbar-item.has-dropdown.is-hoverable:hover .navbar-dropdown {
  display: block !important;
}
```

Now, when hovering over the  `.navbar-item.is-hoverable`, the `display: block` property gets assigned correctly, which makes the dropdown menu show up in IE11 and Edge.
